### DISCUSSION TITLE
Where are the logs stored in ArgoCD component pods? - splunk forwarding use case

### DISCUSSION THREAD

**Discussion by Hariharasuthan99** [argoproj/argo-cd#9202] [Q&A]
Hello,

I am trying to forward logs from ArgoCD pods using Splunk Universal Forwarder as a sidecar container. 

In this use case I will need to mount a shared volume so that the splunk forwarder container can monitor the appropriate .log file created by the argocd conatiners. When I went through the **var/log** path in various argocd cd pods(server, repo-server, etc.) I was unable to find any .log file containing the logs that you can retrieve using '**kubectl logs**' for that particular pod. 

Could someone help with the .log file location for each argocd component pod?

**Comment by rishabh625** [comment]
I think argocd doesn't logs to file, it logs to sysout, u may find logs in docker container folder on Linux host in /var/lib/containers/containerid-json.log

**Reply by Hariharasuthan99** [reply]
Thanks for the reply, is there any documentation on argocd explaining how the logging is implemented?

**Reply by rishabh625** [reply]
1) argocd cli to fetch logs https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_app_logs/

2) https://argo-cd.readthedocs.io/en/stable/operator-manual/security/#logging

**Reply by rishabh625** [reply]
There is no specific that  explains how the logging is implemented?
